lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -545,15 +544,6 @@ def build_module(self, trans, payload={}):
     #
     # -- Helper methods --
     #
-    def _get_tool(self, id, tool_version=None, user=None):
-        id = unquote(id)
-        if tool_version:
-            tool_version = unquote(tool_version)
-        tool = self.app.toolbox.get_tool(id, tool_version)
-        if not tool or not tool.allow_user_access(user):
```

Comment:
This allow_user_access check happens elsewhere?

Reply:
Oh I see it was already unused.

Reply:
Yep, this method is never called since commit 3e3eb4d1397effc8b46c913bb80f3c02773ac506 .